### ISSUE TITLE
Using HERE network

### ISSUE THREAD

**Issue by Dib-AEK** [matsim-org/matsim-code-examples#689]
Hello,

I am a beginner with MATSim and I would like to run some simulations of the Ile de France scenario that you could find [here](https://github.com/eqasim-org/ile-de-france). However, I want to use the traffic network that is provided by [HERE API](https://www.here.com/platform).
I would like to know if anyone has tried to do this before? and how they converted the network provided by HERE API into a MATSim input;

I will highly appreciate any guidance in this regard. Thanks

**Comment by jfbischoff** [comment]
I don't think that the HERE API lets you export network geometries, such as nodes and link locations and attributes (speeds, lanes, etc.).
I could be wrong though. If you manage to get this data in some sensible format from Here, the conversion to a MATSim network could easily be done in either JAVA or Python.

**Comment by Dib-AEK** [comment]
Thank you for responding,
Yes, indeed, I managed to get the traffic network data, but it needs post-processing to convert it into a format understandable by MATSim; that's why I wanted to know if someone had already done that before.
